dts/bindings/gpio/ti,boosterpack-header.yaml

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2019 Brett Witherspoon
 # SPDX-License-Identifier: Apache-2.0
 
-title: TI BoosterPack GPIO header
-
 description: |
+    TI BoosterPack GPIO header
```

Comment:
Maybe this line is redundant with the line after it ("GPIO pins exposed..."), or the information could be merged.